game_event/migrations/0001_initial.py

```diff
@@ -31,7 +31,7 @@ class Migration(migrations.Migration):
                                                         ('Cricket', 'Cricket'), ('Handball', 'Handball')],
                                                max_length=100,
                                                validators=[django.core.validators.MinLengthValidator(4)])),
-                ('court_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='court.court')),
+                ('court', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='court.court')),
             ],
```

Comment:
I belive this changes (update court_id to court) are already been changed in #55.

is there a specific reason you decided you change it again here ?